operators/aqua/2022.4.1/bundle.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=aqua
+LABEL operators.operatorframework.io.bundle.channels.v1=2022.4.0
+LABEL operators.operatorframework.io.bundle.channel.default.v1=2022.4.0
```

Comment:
It shows wrong. You are trying to add the same bundle twice in the same channel.
If you want it the default channel for your package `operators.operatorframework.io.bundle.channel.default.v1=2022.4.0` then you do not need also informed `operators.operatorframework.io.bundle.channels.v1=2022.4.0`.

Also, be aware that you are not using channels that are recommended: https://olm.operatorframework.io/docs/best-practices/channel-naming/

Reply:
We checked that the default channel and channels are mandatory, so it shows right, and this scenario shows fail under the bug found.